platform/persistence/platform-persistence-core-attributes-impl/src/main/java/org/codice/ddf/persistence/attributes/internal/AttributesStoreImpl.java

```diff
@@ -98,7 +120,33 @@ public void setDataUsage(final String username, final long dataUsage)
                         .lock();
 
                 LOGGER.debug("Updating user {} data usage to {}", username, dataUsage);
-                persistentStore.add(PersistentStore.USER_ATTRIBUTE_TYPE, toPersistentItem(username, dataUsage));
+                persistentStore.add(PersistentStore.USER_ATTRIBUTE_TYPE, toPersistentItem(username,
+                        dataUsage,
+                        DEFAULT_DATA_USAGE_LIMIT));
```

Comment:
We want to set the default here, this is the initial entry into the attribute store